packages/amplication-client/src/util/BillingFeature.ts

```diff
@@ -7,6 +7,7 @@ export enum BillingFeature {
   CodePushToGit = "feature-code-push-to-git",
   IgnoreValidationCodeGeneration = "feature-ignore-validation-code-generation",
   HideNotifications = "feature-ignore-validation-code-generation-hide-notifications",
+  AwsCodeCommit = "feature-awscodecommit",
```

Comment:
Did you add it only to the client? I think we should also add it to the server and add a check for the entitlement on the entry of the function, like we did with Bitbucket and import DB

Reply:
Added to the server too. Good catch